### PR TITLE
Release v5.4.1

### PR DESCRIPTION
**Pull request by kyrylo**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,11 @@ Airbrake Changelog
 
 ### master
 
+### [v5.4.1][v5.4.1] (June 21, 2016)
+
+* Fixed Capistrano 3 bug when system has conflicting Rake versions, which would
+  raise `Gem::LoadError` ([#544](https://github.com/airbrake/airbrake/pull/544))
```
Comment (sgray11) [inline CHANGELOG.md]: 564

Comment (shifi) [inline CHANGELOG.md]: This link redirects to issue https://github.com/airbrake/airbrake/issues/544.

Comment (kyrylo) [inline CHANGELOG.md]: Fixed, thanks :)

**Comment by shifi** [comment]
LGTM

**Pull request merged**